Source/JavaScriptCore/ftl/FTLLowerDFGToB3.cpp

```diff
@@ -22595,6 +22595,18 @@ IGNORE_CLANG_WARNINGS_END
         return m_out.phi(Int64, intToInt52, doubleToInt52);
     }
 
+    LValue isStrictInt52(LValue int64Value)
+    {
+        LValue added = m_out.add(m_out.constInt64(0x0008000000000000ULL), int64Value);
```

Comment:
Why `0x0008000000000000ULL` instead of `0xfff8000000000000ULL` as in the original?